org.eclipse.xtext.tests/xtend-gen/org/eclipse/xtext/resource/containers/SourceFolder.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2017 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.xtext.resource.containers;
```

Comment:
maybe move to testing or even to xtext core?

Reply:
Maybe when we need it again...